sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ManageCustomModels.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.AccountProperties;
+import com.azure.ai.formrecognizer.models.CustomFormModel;
+import com.azure.ai.formrecognizer.models.CustomFormModelInfo;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.util.Context;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Sample for demonstrating common custom model management operations.
+ */
+public class ManageCustomModels {
+
+    /**
+     * Main program to invoke the demo for performing operations of a custom model.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(final String[] args) {
+        // Instantiate a client that will be used to call the service.
+        FormTrainingClient client = new FormRecognizerClientBuilder()
+            .apiKey(new AzureKeyCredential("{api_key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildClient().getFormTrainingClient();
+
+        AtomicReference<String> modelId = null;
+
+        // First, we see how many custom models we have, and what our limit is
+        AccountProperties accountProperties = client.getAccountProperties();
+        System.out.printf("The account has %s custom models, and we can have at most %s custom models",
+            accountProperties.getCount(), accountProperties.getLimit());
+
+        // Next, we get a paged list of all of our custom models
+        PagedIterable<CustomFormModelInfo> customModels = client.listModels();
+        System.out.println("We have following models in the account:");
+        customModels.forEach(customFormModelInfo -> {
+            System.out.printf("Model Id: %s%n", customFormModelInfo.getModelId());
+            // get custom model info
+            modelId.set(customFormModelInfo.getModelId());
```

Comment:
I think it'd be better to just print the model ids of the models you're listing, and then call getCustomModel with a single model id and print out more of that information. That way there's less information being printed and you can highlight lisModels and getCustomModel separately